.github/workflows/build-common.yml

```diff
@@ -152,7 +152,7 @@ jobs:
         run: |
           modules=$(ls -d smoke-tests/apps/* | sed 's/\/$//' | sed 's/\//:/g' | sed 's/^/:/')
           inner_json=$(echo $modules | xargs echo | sed 's/ /","/g')
-          echo "::set-output name=matrix::{\"module\":[\"$inner_json\"]}"
+          echo "{matrix}={\"module\":[\"$inner_json\"]}" >> $GITHUB_OUTPUT 
```

Comment:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/